documentation/otel/metrics.md

```diff
@@ -48,23 +50,61 @@ server.route({
   path: '/',
   method: 'GET',
   handler (req, res) {
-    counter.add(1)
+    this.metrics['test-counter'].add(1)
     res.send({ hello: 'world' })
   }
 })
 
-const newrelic = require('newrelic')
-newrelic.agent.on('started', () => {
-  counter = metrics.getMeter('test-meter').createCounter('test-counter')
-  server.listen({ port: 8080 })
+server.decorate('metrics', {
+  'test-counter': metrics.getMeter('test-meter').createCounter('test-counter')
 })
+
+server.listen({ port: 8080 })
 ```
 
 We altered this application in the following ways:
 
 1. We imported the OpenTelemetry API.
+1. We defined our counter, and then started the server.
+
+However, a more robust method would be to wait for the agent to finish
```

Comment:
this doesn't matter anymore does it? since your exporter timeout is higher than our initial connect.